commit b575f8ae9b0faca0fb9b46d369d1776921cd0787 (refs/changes/40/940/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-22T14:56:57-08:00 (2 years, 11 months ago)
    
    joyent/node-manta#288 mfind of file blows assertion: "ent (object) is required"

diff --git a/CHANGES.md b/CHANGES.md
index 76b798e..e2f65dd 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,7 @@
 
 ## not yet released
 
+- joyent/node-manta#288 mfind of file blows assertion: "ent (object) is required"
 - joyent/node-manta#272 Add `--version` to all tools
 - joyent/node-manta#282 `mchmod` now parses all standard options.  The use of
   the `--` form is encouraged to avoid ambiguities in role versus option names
diff --git a/lib/client.js b/lib/client.js
index 4fe45e2..c13be29 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -88,7 +88,7 @@ function InvalidDirectoryError(dir, ent) {
     Error.call(this);
 
     assert.string(dir, 'dir');
-    assert.object(ent, 'ent');
+    assert.optionalObject(ent, 'ent');
 
     this.name = 'InvalidDirectoryError';
     this.message = dir + ' is an invalid manta directory';
